### ISSUE TITLE
System.IO.FileNotFoundException: Could not load file or assembly 'VidCoderFileWatcher, Version=9.1.0.0, Culture=neutral, PublicKeyToken=null'. Das System kann die angegebene Datei nicht finden.

### ISSUE THREAD

**Issue by deltapapa01** [RandomEngy/VidCoder#1125]
### What were you doing when VidCoder crashed?

Code some mkv files
Energy saveing the pc
after restart close vidcoder
crash

### Crash details

```text
VidCoder Beta 9.1 Portable
System.IO.FileNotFoundException: Could not load file or assembly 'VidCoderFileWatcher, Version=9.1.0.0, Culture=neutral, PublicKeyToken=null'. Das System kann die angegebene Datei nicht finden.
File name: 'VidCoderFileWatcher, Version=9.1.0.0, Culture=neutral, PublicKeyToken=null'
   at VidCoder.Services.WatcherProcessManager.SetStatusToStopped()
   at VidCoder.Services.WatcherProcessManager.RefreshStatus()
   at VidCoder.Services.WatcherProcessManager..ctor(IProcesses processes, IAppLogger logger)
   at lambda_method41(Closure , IResolverContext )
   at DryIoc.Factory.<>c__DisplayClass26_1.<ApplyReuse>b__2()
   at DryIoc.Scope.TryGetOrAdd(ImMap`1 items, Int32 id, CreateScopedValue createValue, Int32 disposalOrder)
   at DryIoc.Scope.GetOrAdd(Int32 id, CreateScopedValue createValue, Int32 disposalOrder)
   at DryIoc.Factory.ApplyReuse(Expression serviceExpr, Request request)
   at DryIoc.Factory.GetExpressionOrDefault(Request request)
   at DryIoc.Factory.GetDelegateOrDefault(Request request)
   at DryIoc.Container.ResolveAndCacheDefaultFactoryDelegate(Type serviceType, IfUnresolved ifUnresolved)
   at DryIoc.Container.DryIoc.IResolver.Resolve(Type serviceType, IfUnresolved ifUnresolved)
   at DryIoc.Resolver.Resolve[TService](IResolver resolver, IfUnresolved ifUnresolved)
   at VidCoder.ViewModel.MainViewModel.OnClosing()
   at VidCoder.Services.Windows.WindowManager.OnClosing(Window window, Boolean userInitiated)
   at VidCoder.Services.Windows.WindowManager.OnClosingHandler(Object sender, CancelEventArgs e)
   at System.Windows.Window.OnClosing(CancelEventArgs e)
   at System.Windows.Window.WmClose()
   at System.Windows.Window.WindowFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.PublicHooksFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by RandomEngy** [comment]
I'm not sure why that's happening... even on the Portable version that assembly should be there, but also there's no reason we need to be invoking that code path in the portable version. I added another check to only run this code when the watcher is supported and enabled, which should prevent this from happening. Will be in the next Beta.

**Comment by RandomEngy** [comment]
Fixed in 9.4 Beta.
